### PR TITLE
fix: [PL-28300]: group search error handler with hint and suggestion

### PR DESCRIPTION
**Pull request by pbarapatre10**
#### Screenshots

Before:
image

After:
image

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Passed_** 
 


            
 #### 1 Issues 
* Code issue 1 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/11331)

**Comment by pbarapatre10** [comment]
retrigger jest

File: src/modules/20-rbac/modals/LinkToSSOProviderModal/views/LinkToLDAPProviderForm.tsx [hunk 1]
```diff
@@ -99,7 +105,11 @@ const LinkToLDAPProviderForm: React.FC = () => {
       <Text font={{ variation: FontVariation.SMALL_SEMI }} color={Color.GREY_400}>
         {getString('rbac.userDetails.linkToSSOProviderModal.groupSearchInfo')}
       </Text>
-      {errorGroupSearch && <PageError message={getRBACErrorMessage(errorGroupSearch as any)} />}
+      {responseMessages && (
+        <Layout.Vertical>
+          <ErrorHandler responseMessages={responseMessages} className={css.layoutErrorMessage} />
```
Comment (upadhyay-an) [inline src/modules/20-rbac/modals/LinkToSSOProviderModal/views/LinkToLDAPProviderForm.tsx:110]: Do we need this CSS class?

Comment (pbarapatre10) [reply src/modules/20-rbac/modals/LinkToSSOProviderModal/views/LinkToLDAPProviderForm.tsx:110]: Yes @upadhyay-an , without that the margin wasn't getting added

**Review by upadhyay-an — APPROVED** [review]

File: src/modules/20-rbac/modals/LinkToSSOProviderModal/views/LinkToLDAPProviderForm.tsx [hunk 2]
```diff
@@ -99,7 +105,11 @@ const LinkToLDAPProviderForm: React.FC = () => {
       <Text font={{ variation: FontVariation.SMALL_SEMI }} color={Color.GREY_400}>
         {getString('rbac.userDetails.linkToSSOProviderModal.groupSearchInfo')}
       </Text>
-      {errorGroupSearch && <PageError message={getRBACErrorMessage(errorGroupSearch as any)} />}
+      {responseMessages && (
+        <Layout.Vertical>
```
Comment (meenakshi-harness) [inline src/modules/20-rbac/modals/LinkToSSOProviderModal/views/LinkToLDAPProviderForm.tsx:109]: Do we need this?

Comment (pbarapatre10) [reply src/modules/20-rbac/modals/LinkToSSOProviderModal/views/LinkToLDAPProviderForm.tsx:109]: Yes, without that the margin is not getting applied.

Comment (meenakshi-harness) [reply src/modules/20-rbac/modals/LinkToSSOProviderModal/views/LinkToLDAPProviderForm.tsx:109]: Doesn't seem right. This is just a div we are adding. Approving for now, we can check later.

**Review by meenakshi-harness — APPROVED** [review]

**Pull request merged**